olaf/scripts/runner.py

```diff
@@ -0,0 +1,43 @@
+import os
+import spacy
+from abc import ABC, abstractmethod
+from olaf import Pipeline
+from olaf.repository.corpus_loader import TextCorpusLoader
+
+
+class Runner(ABC):
+    def __init__(self, model_name="en_core_web_md", corpus_file=""):
+        """Initialise a pipeline Runner.
+
+        Attributes
+        ----------
+        pipeline: Pipeline
+            The pipeline to execute.
+        """
+        spacy_model = spacy.load(model_name)
+        if os.path.isfile(corpus_file):
+            corpus_loader = TextCorpusLoader(corpus_path=corpus_file)
+        elif corpus_path := os.path.isfile(
+            os.path.join(os.getenv("DATA_PATH"), corpus_file)
+        ):
+            corpus_loader = TextCorpusLoader(corpus_path=corpus_path)
+        else:
+            corpus_path = os.path.isfile(
+                os.path.join(os.getenv("DATA_PATH"), "pizza_description.txt")
+            )
+            corpus_loader = TextCorpusLoader(corpus_path=corpus_path)
+        self.pipeline = Pipeline(spacy_model=spacy_model, corpus_loader=corpus_loader)
```

Comment:
Redundant with the `no_llm_pipeline.py` init. Either you remove the code from here or in the `no_llm_pipeline.py` init